app/scripts/sagan/sim.js

```diff
@@ -34,7 +34,23 @@
 
   Sim.prototype.makeSkybox = function() {
     this._skyGeometry = new THREE.CubeGeometry(49999,49999,49999);
-    this._skyMaterial = new THREE.MeshBasicMaterial({color: 0x0B1317, side: THREE.BackSide});
+    //this._skyMaterial = new THREE.MeshBasicMaterial({color: "red", side: THREE.BackSide});
+    var skyFace; //object for face
+    var faceMaterialArray= [];
```

Comment:
Should follow the style of the rest of the code with a space around the equal sign.
`var faceMaterialArray = [];`